roles/zuul/files/jobs/lore.yaml

```diff
@@ -6,8 +6,9 @@
       - zuul-git-prep
       - shell: |
           #!/bin/bash -ex
+          sudo apt-get install -y curl
           curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
-          sudo apt-get install -qq nodejs
+          sudo apt-get install -y nodejs
```

Comment:
At this point, seeing it might not be terrible. We can always turn it back off once this has settled.